stratuslab/python/tar/slipstream_stratuslab/StratusLabIterClientCloud.py

```diff
@@ -93,6 +92,19 @@ def _start_image_for_deployment(self, node_instance, vm_name):
             raise
         return runner
 
+    def _ensure_pdisk_endpoint_is_set(self):
```

Comment:
In fact, we need to get rid of this env var, and instead provide pdisk endpoint as CLI param. Created #22. Until #22 is fixed, I'll use what you are proposing.